docs/reference/manifest/desktopformfactor.md

```diff
@@ -4,18 +4,14 @@ Specifies the settings for an add-in for the desktop form factor. The desktop fo
 
 Each DesktopFormFactor definition contains the  **FunctionFile** element and one or more **ExtensionPoint** elements. For more information, see [FunctionFile element](functionfile.md) and [ExtensionPoint element](extensionpoint.md).
 
-> [!IMPORTANT]
-> The SupportsSharedFolders element is only available in the Outlook add-ins Preview Requirement Set against Exchange Online.
-> Add-ins that use this element aren't allowed in the Office Store or Centralized Deployment.
-
 ## Child elements
 
 | Element                               | Required | Description  |
 |:--------------------------------------|:--------:|:-------------|
 | [ExtensionPoint](extensionpoint.md)   | Yes      | Defines where an add-in exposes functionality. |
 | [FunctionFile](functionfile.md)       | Yes      | A URL to a file that contains JavaScript functions.|
 | [GetStarted](getstarted.md)           | No       | Defines the callout that appears when installing the add-in in Word, Excel, or PowerPoint hosts. |
-| SupportsSharedFolders                 | No       | Defines whether the Outlook add-in is available in delegate scenarios and is set to *false* by default. Preview requirement set.|
+| [SupportsSharedFolders](supportssharedfolders.md) | No | Defines whether the Outlook add-in is available in delegate scenarios and is set to *false* by default. Preview requirement set.<br><br>**Important**: This element is only available in the Outlook add-ins Preview Requirement Set against Exchange Online. Add-ins that use this element aren't allowed in the Office Store or Centralized Deployment. |
```

Comment:
Suggest that you remove the first mention of "Preview requirement set." -- since this information is included in the note that you added.  Also, suggest rephrasing the final sentence of the note to:

Add-ins that use this element cannot be published to AppSource or deployed via centralized deployment.